metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/lineage.py

```diff
@@ -752,8 +752,17 @@ def _create_lineage_map(
 
             # Try the sql parser first.
             if self.config.lineage_use_sql_parser:
+                if e.statementType == "SELECT":
+                    # We wrap select statements in a CTE to make them parseable as insert statement.
+                    # This is a workaround for the sql parser to support the case where the user runs a query and inserts the result into a table..
+                    query = f"""create table {destination_table.table_identifier.get_table_name()} AS
```

Comment:
do we need quotes e.g.

```suggestion
                    query = f"""create table `{destination_table.table_identifier.get_table_name()}` AS
```